.github/workflows/main.yaml

```diff
@@ -22,6 +22,15 @@ jobs:
       - name: Test
         run:  make test
       - name: Build
-        run:  make build
+        run:  make cross-platform
       - name: Lint
         run: make check
+      - name: Permissions
+        run: chmod a+x faas_*amd*
+      - uses: actions/upload-artifact@v2
+        with:
+          name: Binaries
+          path: |
+            faas_darwin_amd64
+            faas_linux_amd64
+            faas_windows_amd64.exe
```

Comment:
Regarding your question about separate downloads; from what I can see from how actions are used in other places, the way to upload each binary individually would be to repeat this stanza three times.  Each named something like "Windows Pre-release Binary"... or something like that.

Reply:
Testing this now

Reply:
You are correct!

![image](https://user-images.githubusercontent.com/15952/92252787-b956be00-ee9c-11ea-9ee1-d0f3f488a19f.png)